composer/rest/composer2/composer2_airflow_rest_api_test.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+from unittest import mock
+
+import pytest
+
+import composer2_airflow_rest_api
+
+COMPOSER2_WEB_SERVER_URL = os.environ["COMPOSER2_WEB_SERVER_URL"]
+DAG_CONFIG = {"test": "value"}
+
+
+@pytest.fixture(scope="function")
+def successful_response():
+    response_mock = mock.Mock()
```

Comment:
Any reason you didn't use an [autospec](https://docs.python.org/3/library/unittest.mock.html#auto-speccing) mock here? Autospeccing can help catch errors where you misspelled a property name, for example.

```suggestion
    response_mock = mock.create_autospec(requests.Response, instance=True)
```

Reply:
lack of experience with mocks! no good reason. Thank you for the helpful link!